docs/install/collecting-metrics/README.md

```diff
@@ -0,0 +1,97 @@
+This document describes how to set up the
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/about/) to
+receive metrics from the Knative infrastructure components and distribute them
+to Prometheus. [OpenTelemetry](https://opentelemetry.io/) is a CNCF project to
+provide consistent interfaces for instrumenting applications and platforms to
+provide telemetry information for developers and operators. OpenTelemetry allows
+Knative to build provider-agnostic instrumentation into the platform, so that
+it's easy to export logs to multiple monitoring services without needing to
+rebuild or reconfigure the Knative binaries.
+
+## Setting up the collector
+
+The collector provides a long-lived location where various Knative components
+can push metrics (and eventually traces) to be retained and collected by a
+monitoring service. For this example, we'll configure a single collector
+instance using a ConfigMap and a Deployment. For more complex deployments, some
+of this can be automated using the
+[opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-operator),
+but it's also easy to manage this service directly.
+
+![Diagram of components reporting to collector, which is scraped by Prometheus](./system-diagram.svg)
```

Comment:
I think we need to make clear that this is an example architecture and provide a link to more options [here](https://github.com/open-telemetry/opentelemetry-collector/blob/master/docs/design.md).
Some users might want to use an Collector as an Agent etc, do we support that?

Reply:
I think those should work; I wanted to include a minimal example here.